### DISCUSSION TITLE
I don't know why my vlad is slower than A1111 with the same device

### DISCUSSION THREAD

**Discussion by tzwm** [vladmandic/sdnext#1501] [Q&A]
#### vlad
<img width="1280" alt="01170aa746dcf37f6bace291a6a43412e6bac2fe1915105ae8f66a103c71160b" src="https://github.com/vladmandic/automatic/assets/828837/7abec606-ef94-4daf-bca3-829e744e6ca8">

#### a1111
<img width="1263" alt="image" src="https://github.com/vladmandic/automatic/assets/828837/3b86b515-9441-4718-8f13-8236b2843ea2">

**Comment by brknsoul** [comment]
That's because you're not using xformers on SD.Next.
Settings > Optimizations > xformers.

EDIT: Rather, I should say, these comparisons are faulty, since they're not using the same settings.

**Comment by vladmandic** [comment]
and it could be a number of other things - check live preview, token merging, etc...
really comparing screenshots of benchmarks without any data is not going to be of much help.
